Tests/MapboxMapsTests/Snapshot/MapboxMapsSnapshotTests.swift

```diff
@@ -187,4 +187,72 @@ class MapboxMapsSnapshotTests: XCTestCase {
         wait(for: [expectation], timeout: 10.0)
     }
 
+    func testShowsLogo() throws {
```

Comment:
There's a lot of duplicated code between this method and `testDoNotShowsLogo`. Let's refactor it to be less repetitive.